host/src/CoreToolsHost/Program.cs

```diff
@@ -47,11 +47,10 @@ static async Task Main(string[] args)
 
         private static bool ElementExistsWithValue(JsonElement element, string key, string value)
         {
-
             return !string.IsNullOrEmpty(key) && !string.IsNullOrEmpty(value)
                 && element.TryGetProperty(key, out JsonElement property) 
-                && !string.IsNullOrEmpty(property.GetString()) 
-                && property.GetString() == value;
+                && !string.IsNullOrEmpty(property.ToString()) 
```

Comment:
This line could probably be removed, but I'm happy to leave it here just for extra defensiveness.